TriblerGUI/widgets/trustpage.py

```diff
@@ -85,11 +85,16 @@ def __init__(self):
         self.statistics = None
         self.blocks = None
         self.byte_scale = 1024 * 1024
+        self.timer = None
```

Comment:
Could you please rename this variable to something more descriptive? Like `refresh_timer`?